homeassistant/components/flexit_bacnet/binary_sensor.py

```diff
@@ -0,0 +1,78 @@
+"""The Flexit Nordic (BACnet) integration."""
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from flexit_bacnet import FlexitBACnet
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import FlexitCoordinator
+from .const import DOMAIN
+from .entity import FlexitEntity
+
+
+@dataclass(kw_only=True, frozen=True)
+class FlexitBinarySensorEntityDescription(BinarySensorEntityDescription):
+    """Describes a Flexit binary sensor entity."""
+
+    value_fn: Callable[[FlexitBACnet], bool]
+
+
+SENSOR_TYPES: tuple[FlexitBinarySensorEntityDescription, ...] = (
+    FlexitBinarySensorEntityDescription(
+        key="electric_heater",
+        device_class=BinarySensorDeviceClass.HEAT,
+        translation_key="electric_heater",
```

Comment:
Is this the aux heat?

Reply:
It is a sensor telling if the aux heat is on or off.